SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.10.x     | :white_check_mark: |
+| < 2.10.x   | :x:                |
```

Comment:
Should we support the previous version, at least for some time after releasing the new one?

Reply:
Upgrading from a minor to another should not be a big deal, so it may be best to simply ask people to upgrade to the next minor to get the security fix.

Reply:
In any case, the current content reflects the current policy (minus maintaining support for 1.8.x, which maybe we should stop). So I think we could merge as is and discuss changing the policy separately, e.g. maybe having some LTS version or something.